upgrade_guide/README.md

```diff
@@ -1,5 +1,15 @@
 ## Upgrade guide
 
+### 4.14.0
+
+#### AllNodesFailedException instead of NoNodeAvailableException in certain cases
+
+[JAVA-2959](https://datastax-oss.atlassian.net/browse/JAVA-2959) changed the behavior for when a
+request is directed to a particular node (via `Statement#setNode(Node)`) and all connections to that
+node are busy. Previously you would get back a `NoNodeAvailableException` but you will now get back
+an `AllNodesFailedException` where the `getAllErrors` map contains a `NodeUnavailableException`
+for that node.
```

Comment:
`Statement#setNode(Node)` is not the only way by which you could observe the new behavior. I suggest a more generic rephrasing:

```suggestion
[JAVA-2959](https://datastax-oss.atlassian.net/browse/JAVA-2959) changed the behavior for when a
request cannot be executed because all nodes tried were busy. Previously you would get back a 
`NoNodeAvailableException` but you will now get back an `AllNodesFailedException` where the 
`getAllErrors` map contains a `NodeUnavailableException` for that node.
```

Reply:
done